modules/loki-alerts.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * logging_alerts/default-logging-alerts.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="loki-alerts_{context}"]
+= {loki-op} alerts
+
+The following alerts are generated by the {loki-op}. You can view these alerts in the {ocp-product-title} web console.
+
+.{loki-op} alerts
+[cols="2,2,2,1",options="header"]
+|===
+|Alert |Message |Description |Severity
+
+|`LokiRequestErrors`
+|`{{ $labels.job }} {{ $labels.route }} is experiencing <value>% errors.`
+|At least 10% of requests are responded by 5xx server errors.
+|critical
+
+|`LokiStackWriteRequestErrors`
+|`<value>% of write requests from {{ $labels.job }} in <namespace> are returned with server errors.`
+|At least 10% of write requests to the lokistack-gateway are responded with 5xx server errors.
+|critical
+
+|`LokiStackReadRequestErrors`
+|`<value>% of query requests from {{ $labels.job }} in <namespace> are returned with server errors.`
+|At least 10% of query requests to the lokistack-gateway are responded with 5xx server errors.
+|critical
+
+|`LokiRequestPanics`
+|`{{ $labels.job }} is experiencing an increase of <value> panics.`
+|A panic was triggered.
+|critical
+
+|`LokiRequestLatency`
+|`{{ $labels.job }} {{ $labels.route }} is experiencing <value>s 99th percentile latency.` 
+|The 99th percentile is experiencing latency higher than 1 second.
```

Comment:
Per the [Red Hat Technical Writing Style Guide](https://stylepedia.net/style/#numbers):  Spell out ... numbers zero through nine...
```suggestion
|The 99th percentile is experiencing latency higher than one second.
```